Misc/NEWS.d/next/C API/2022-09-15-15-21-34.gh-issue-96803.ynBKIS.rst

```diff
@@ -0,0 +1,9 @@
+Expose C-API functions to get the code object, lasti and line number from
+the internal ``_PyInterpreterFrame`` in the limited API. The functions are:
```

Comment:
```suggestion
the internal ``_PyInterpreterFrame`` as private functions in the public API. The functions are:
```

Reply:
Our terminology is awful. If it is private, then it isn't part of an API.

Can we use the term "unstable" or is that prohibited until PEP 689 is accepted?

Reply:
Yeah, I find all of this confusing as well. Let's use "unstable"

Reply:
The current C API documentation is even worse :-)

> Names beginning with _Py are for **internal use** by the Python implementation and **should not be used** by extension writers.

https://docs.python.org/dev/c-api/intro.html#include-files

See also the open discuss: https://discuss.python.org/t/c-api-what-should-the-leading-underscore-py-mean/18486

Yeah, naming things in the C API is a Work-In-Progress and it evolves a lot in the last years.

Reply:
We have PEP 689 now, so these will be part of the unstable API.